packages/client-app/src/components/AddressBar/AddressBarServer.vue

```diff
@@ -36,6 +36,20 @@ const updateSelectedServer = (serverUid: string) => {
 const isSelectedServer = (serverId: string) => {
   return activeCollection.value?.selectedServerUid === serverId
 }
+
+// Replace variables in URL
+const serverUrl = computed(() => {
+  const server = servers[activeCollection.value?.selectedServerUid ?? '']
+  const url = server?.url
+
+  // TODO: use `replaceVariables` from `@scalar/oas-utils/helpers`
+  // Note: Currently, it’s in @scalar/api-client, but that’s about to change.
+  const singleCurlyBrackets = /{\s*([\w.-]+)\s*}/g
+  return url.replace(singleCurlyBrackets, (match, key) => {
+    const variable = server?.variables?.[key]
+    return variable?.value || variable?.default || variable?.enum?.[0] || match
+  })
+})
```

Comment:
This will error at runtime when the selectedServerUid does not exist in the array.

I would suggest:
```ts
if(!servers.length) return ''
const server = servers[activeCollection.value?.selectedServerUid ?? ''] ?? servers[0]
const url = server.url
```

or  the following which will correctly assign the type to URL

```ts
const url = server?.url as string | undefined
```